build-parent/pom.xml

```diff
@@ -95,10 +95,9 @@
         <junit4.version>4.13.2</junit4.version>
 
         <!-- The image to use for tests that run Keycloak -->
-        <!-- IMPORTANT: If this is changed you must also update bom/application/pom.xml and KeycloakBuildTimeConfig/DevServicesConfig in quarkus-oidc/deployment to match the version -->
-        <keycloak.version>25.0.6</keycloak.version>
+        <keycloak.test.server.version>25.0.6</keycloak.test.server.version>
```

Comment:
Is the idea to keep the test server and the client versions separate?

Reply:
Yes, this is it. I have changed the keycloak.version property to keycloak-client.version (it can be released asynchronous way with regards to Keycloak Server).
keycloak.test.server.version property determines Keycloak Server (Container) to be used.

Reply:
@pskopek Why do we have introduce this separation?

Reply:
@pskopek May be I'm missing something but if start introducing different values for these versions then we can quite quickly get into into some incompatibilities between these client libraries and the Keycloak version we use for testing. 
Can we have them aligned in Quarkus with the update to KC 26.0.0, possibly after this update goes in ?

Reply:
Also, `keycloak.server.version` is a bit better, as it is also used in devmode for developing,  not only testing

Reply:
@sberyozkin the separation is need as keycloak-client libraries are supposed to have different release cycle than keycloak server. Keycloak Client Libs will always support two latest major server versions. (at the moment actually three 24.0.z, 25.0.z and 26.0.z)

Reply:
@pskopek do we know what typically will be released first: the Keycloak client or server?

Reply:
That is an interesting idea, as downstream the KC server version can be different. 
Keeping the same versions upstream for both the client and server seems to be the option which is least likely to cause confusion or side effects

Reply:
So are we aligning the versions in this repo on the server one?

Reply:
The server version is declared here and keycloak-client is located at [bom](https://github.com/quarkusio/quarkus/blob/9b3fb7f44bd4c731348e74719e0d127964d4a3e0/bom/application/pom.xml#L186). At the moment 26.0.0.

Reply:
We would set the property in https://github.com/quarkusio/quarkus/blob/main/pom.xml#L70 in this case